addons/account/models/account_report.py

```diff
@@ -434,13 +434,16 @@ def _compute_auditable(self):
     def _get_auditable_engines(self):
         return {'tax_tags', 'domain', 'account_codes', 'external', 'aggregation'}
 
+    def _strip_formula(self, vals):
+        if 'formula' in vals and isinstance(vals['formula'], str):
+            vals['formula'] = re.sub(r'\s', '', vals['formula'])
```

Comment:
Doing it like this, your replacing spaces, tabs and \n in the whole formula, right ? I'd avoid that; some formulas might want to support spaces within them. Moreover, for someone reading them in the UI, keeping the spaces between the operators makes them more readable.

Let's stick to a mere strip() and replace \n (for them, we're sure we don't want them in any case; they won't even be rendered in the UI anyway)

Reply:
Yes it is replacing spaces, \t and \n. I can't think of a situation where space in formula would be meaningful.
But ok, just note that if we do that
`<field name="aggregate_formula">
        abc 
        + def
</field>`
will be rendered as 'abc        + def'. 
So not exactly readable either.

Reply:
Hmmm good point. How about something like

re.sub(r'\s{2,}', ' ', formula.strip())

?

Reply:
And \t also needs to be replaced, for the same reasons as \n.
![image](https://user-images.githubusercontent.com/107402834/199949819-27616910-6cb3-412f-b63c-913747c99195.png)

Reply:
re.sub(r'\s+', ' ', formula.strip())

Should do the job. Basically, we replace everything by spaces, and only keep one.

Reply:
I like this solution, I'll push changes soonTM